README.md

```diff
@@ -54,6 +54,7 @@ $ make
 Copy the compile ZNC module to your ZNC settings:
 
 ```bash
+$ mkdir ~/.znc/modules
```

Comment:
Perhaps better to use `mkdir -p`, the `-p` option will mean that mkdir will also create the `.znc` directory if it does not exist.